docs/classes/getusermediaerror.html

```diff
@@ -112,7 +112,7 @@ <h3>Properties</h3>
 								<li class="tsd-kind-property tsd-parent-kind-class"><a href="getusermediaerror.html#cause" class="tsd-kind-icon">cause</a></li>
 								<li class="tsd-kind-property tsd-parent-kind-class tsd-is-inherited"><a href="getusermediaerror.html#message" class="tsd-kind-icon">message</a></li>
 								<li class="tsd-kind-property tsd-parent-kind-class tsd-is-inherited"><a href="getusermediaerror.html#name" class="tsd-kind-icon">name</a></li>
-								<li class="tsd-kind-property tsd-parent-kind-class tsd-is-overwrite tsd-is-inherited"><a href="getusermediaerror.html#stack" class="tsd-kind-icon">stack</a></li>
+								<li class="tsd-kind-property tsd-parent-kind-class tsd-is-inherited"><a href="getusermediaerror.html#stack" class="tsd-kind-icon">stack</a></li>
```

Comment:
Why is there doc change?

Reply:
No idea, it appears after running `npm run build:release` to generate the doc.

Reply:
Can you track down what change causes it?

Reply:
It's due to the type change of Error. Because we update the `@types/node`.

Reply:
Btw do you think upgrading this `@types/node` is a breaking change?